test/juvix.yaml

```
@@ -2,7 +2,7 @@ dependencies:
 - ../
 - git:
     url: https://github.com/anoma/juvix-quickcheck.git
-    ref: update-for-0.5.0
+    ref: main
```

Use a release tag instead of main

```suggestion
    ref: v0.7.0
```